packages/embark/src/lib/core/processes/processManager.js

```diff
@@ -65,6 +69,42 @@ class ProcessManager {
     return processList;
   }
 
+  _registerCommands() {
+    // do not allow whisper service to be started/stopped as it requires a restart of embark
+    const availableProcesses = Object.keys(this.processes).filter((name) => !["whisper", "embark"].includes(name.toLowerCase()));
```

Comment:
We most likely have already some constants for whisper and embark process names. Maybe we can take advantage of them here? 

Not a biggie, tho